definitions/modules/transponder.yaml

```diff
@@ -2,7 +2,29 @@ shared:
   -
     type: var
     var_name: A:TRANSPONDER STATE:1
-    var_units: number
+    var_units: Enum
+    var_type: i32
+  -
+    type: NumDigitSet
+    var_name: A:TRANSPONDER CODE:1
+    var_units: Number
+    var_type: i32
+    up_event_names:
+      - XPNDR_1_INC
+      - XPNDR_10_INC
+      - XPNDR_100_INC
+      - XPNDR_1000_INC
+    down_event_names:
+      - XPNDR_1_DEC
+      - XPNDR_10_DEC
+      - XPNDR_100_DEC
+      - XPNDR_1000_DEC
+
+init:
```

Comment:
same as above

Reply:
Added because of transponder placed under init on 32NX profile.
https://github.com/Sequal32/yourcontrols/blob/master/definitions/aircraft/FlyByWire_A32NX.yaml#L1404

Reply:
In that case, the transponder should not be under shared for the A32NX as the transponder uses H events that will throw off the sync. It'll need to be defined individually for it.